packages/core/src/domain/sessionManagement.ts

```diff
@@ -33,11 +33,13 @@ export function startSessionManagement<TrackingType extends string>(
   const sessionCookie = cacheCookieAccess(SESSION_COOKIE_NAME, options)
   tryOldCookiesMigration(sessionCookie)
   const renewObservable = new Observable<void>()
-  let currentSessionId = retrieveActiveSession(sessionCookie).id
+  const sessionState = retrieveActiveSession(sessionCookie)
+  let initialSessionState = { ...sessionState }
```

Comment:
`sessionState` is not used anywhere else, and `initialSessionState` is not mutated, you could use a single variable for this.